examples/pytorch/test_accelerate_examples.py

```diff
@@ -318,7 +317,7 @@ def test_run_image_classification_no_trainer(self):
             --learning_rate 1e-4
             --per_device_train_batch_size 2
             --per_device_eval_batch_size 1
-            --max_train_steps 2
+            --max_train_steps 10
```

Comment:
I'd prefer we update the expected values in tests than increased the number of train steps, as this will increase the running time for tests

Reply:
Okay, we can do that. This is in-line with the number of train steps in the pytorch example tests, but I can modify it to do 2 steps instead!

Reply:
Changed thresh to be 0.4, what I observed the final result to be.